boto/connection.py

```diff
@@ -1112,6 +1115,7 @@ def make_request(self, action, params=None, path='/', verb='GET'):
             http_request.params['Action'] = action
         if self.APIVersion:
             http_request.params['Version'] = self.APIVersion
+        statsd.increment('boto.request', tags=["action:%s" % action or 'unknown', "host:"])
```

Comment:
If we want to split by role & co, we'd need the host tag. I'll remove this `host:`, we could still add it back if there's too many contexts for a good usability